src/utils/buildProject.js

```diff
@@ -0,0 +1,54 @@
+import axios from 'axios';
+
+const MAX_CONTENT_LENGTH = 100 /* MB */ * 1024 * 1024;
+
+export default async function buildApkAsync(
+  appJson : Object,
+  headers : Object,
+  options : Object
+) {
+
+  const exp = appJson.expo;
+
+  return await _callMethodAsync('build', 'put', headers, {
+    manifest: exp,
+    options : options,
+  });
+}
+
+async function _callMethodAsync(
+  methodName: string,
+  method: string,
+  headers: Object,
+  requestBody: Object
+): Promise<any> {
+
+  let url =
+      'http://0.0.0.0:3000/--/api/' +
```

Comment:
this should use https://github.com/expo/snack-sdk/blob/b390ad9716fc5e1e09aa84cd2ae40dce37fb19ce/src/SnackSession.js#L140